### PR TITLE
feat(CustomerQuoteCard): Create CustomerQuoteCard component

### PR DESCRIPTION
**Pull request by lialila**
Story: https://trello.com/c/xR1oOYCf/404-customer-quote-card-3
Preview: https://storybook-n2hbd8wc7-treely.vercel.app/?path=/story/slices-customerstories--with-customer-card

### CODE REVIEW

File: src/models/strapi/StrapiCustomerStory.ts [hunk 1]
```diff
@@ -5,19 +5,22 @@ import StrapiMetadata from './StrapiMetadata';
...
+  cardImage?: StrapiImage;
+  quoteCardQuote?: string;
   locale: Locale;
   createdAt: string;
   updatedAt: string;
   metadata: StrapiMetadata | null;
   slices: any[];
   localizations: StrapiLocalization[];
   topBanner?: StrapiTopBanner;
+  variant?: string;
```
Comment (rschiffer) [inline src/models/strapi/StrapiCustomerStory.ts:23]:
```suggestion
  variant?: 'customerCard' | 'quoteCard';
```

File: src/rootMessages.en.ts [hunk 2]
```diff
@@ -27,12 +29,14 @@ const rootMessagesEn = {
   //
   // Slices
   //
+  ...glossaryMessagesEn,
```
Comment (rschiffer) [inline src/rootMessages.en.ts:32]: Is there a reason for breaking up the alphabetical order?

File: src/slices/CustomerStories/CustomerStories.test.tsx [hunk 3]
```diff
@@ -1,44 +1,36 @@
...
     expect(
-      screen.getByText(strapiCustomerStoryMock.attributes.title)
+      screen.getByText(strapiCustomerStoryMock.attributes.title || '')
     ).toBeInTheDocument();
   });
 
   it('displays nothing if the customer story that should be displayed does not exists in the delivered customer stories', () => {
-    setup({ customerStories: [] });
+    const slice = {
+      customer_stories: [],
```
Comment (rschiffer) [inline src/slices/CustomerStories/CustomerStories.test.tsx:28]:
```suggestion
      customerStories: [],
```
Is there a reason for not using camel case?

File: src/slices/QuoteCustomerCard/QuoteCustomerCard.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import { Container, Heading, Button, Link, Box, Text } from 'boemly';
+import React, { useContext } from 'react';
+import { StrapiCustomerStory, strapiMediaUrl } from '../..';
+import Image from 'next/image';
+import { IntlContext } from '../../components/ContextProvider';
+
+export interface QuoteCustomerCardProps {
+  customerStory: StrapiCustomerStory;
+}
+export const QuoteCustomerCard = ({
```
Comment (rschiffer) [inline src/slices/QuoteCustomerCard/QuoteCustomerCard.tsx:10]:
```suggestion
export const CustomerQuoteCard = ({
```

What do you think? Just a suggestion.

File: src/slices/CustomerStories/CustomerStories.test.tsx (hunk 3 above)
Comment (lialila) [reply src/slices/CustomerStories/CustomerStories.test.tsx:28]: I don't know the reason why we don't use camel case here, it was done this way before this story. My idea was not to rebuild the concepts. What do you think?

**Comment by lukasbals** [comment]
The commit message has not the right format ⚠️

**Pull request merged**